src/sub-pages/_single.html

```diff
@@ -55,13 +55,13 @@
         </section>
         {% endif %}
 
-        {% set content = { 'markup' : sub_page.body_content } %}
+        {% set content = { 'markup' : sub_page.body_content|safe } %}
```

Comment:
Update to `sub_page.body_content | safe` here and L:64 for consistency in change made in `src/offices/_single.html`